zilencer/management/commands/populate_db.py

```diff
@@ -327,18 +327,14 @@ def handle(self, **options: Any) -> None:
                 do_change_plan_type(mit_realm, Realm.SELF_HOSTED, acting_user=None)
                 RealmDomain.objects.create(realm=mit_realm, domain="mit.edu")
 
-                lear_realm = Realm.objects.create(
+                lear_realm = do_create_realm(
                     string_id="lear",
                     name="Lear & Co.",
                     emails_restricted_to_domains=False,
                     invite_required=False,
                     org_type=Realm.CORPORATE,
                 )
-                RealmAuditLog.objects.create(
-                    realm=lear_realm,
-                    event_type=RealmAuditLog.REALM_CREATED,
-                    event_time=lear_realm.date_created,
-                )
+                do_change_plan_type(lear_realm, Realm.SELF_HOSTED, acting_user=None)
```

Comment:
Can you move the non-zulip realm commits earlier in the branch?  It seems like we'll want to merge those before the `zulip` one, since that one has all the (testing, etc.) complexity.

Reply:
Yup. Done.